src/rules/no-unused-modules.js

```diff
@@ -347,9 +350,9 @@ module.exports = {
           context.report(
             node,
             `exported declaration '${value}' not used within other modules`
-          )
-        }
-      } else {
+            )
```

Comment:
this indentation is wrong
```suggestion
          )
```